modules/cli/src/main/resources/completions/zsh

```diff
@@ -54,39 +59,35 @@ _coursier() {
   local scalaVersion=( {--scala-version,-e}'[default Scala version]:version:' )
   local sourcesR=( {--sources,-R}'[source repository; for multiple repositories, separate with comma and/or add this option multiple times]:value:' )
   local sourcesS=( {--sources,-S}'[fetch source artifacts]' )
-  local sparkAssemblyDependencies=( '--spark-assembly-dependencies[Spark assembly dependencies]:value:' )
-  local sparkVersion=( '--spark-version[Spark version]:version:' )
   local standalone=( {--standalone,-s}'[generate a standalone launcher, with all JARs included, instead of one downloading its dependencies on startup]' )
   local submitDependencies=( '--submit-dependencies[submit dependencies]:value:' )
   local target=( {--target,-d}'[native compilation target directory]:value:' )
   local tree=( {--tree,-t}'[print dependencies as a tree]' )
   local ttl=( {--ttl,-l}'[TTL duration, e.g. "24 hours"]:duration:' )
   local typelevel=( '--typelevel[swap the mainline Scala JARs by Typelevel ones]' )
   local verbose=( {--verbose,-v}'[increase verbosity; specify several times to increase more]' )
-  local yarnIdFile=( '--yarn-id-file[if master is yarn-cluster, write YARN app ID to a file]' )
-  local yarnVersion=( '--yarn-version[YARN version; only used with Spark 2]' )
 
   case $words[1] in
     bootstrap)
       _arguments -C $artifactType $force $mainClass $utput $force $standalone $property $javaOpt $native $target $keepTarget $isolated $isolateTarget $keepOptional $mode $ttl $quiet $verbose $progress $maxIterations $repository $sourcesR $noDefault $sbtPluginHack $dropInfoAttr $forceVersion $exclude $scalaVersion $intransitive $classifier $defaultConfiguration $parallel $checksum $benchmark $tree $reverseTree $profile $typelevel $cache
       ;;
 
-    launch)
-      _arguments -C $mainClass $extraJars $isolated $isolateTarget $keepOptional $mode $ttl $quiet $verbose $progress $maxIterations $repository $sourcesR $noDefault $sbtPluginHack $dropInfoAttr $forceVersion $exclude $scalaVersion $intransitive $classifier $defaultConfiguration $parallel $checksum $benchmark $tree $reverseTree $profile $typelevel $cache
-      ;;
-    
     fetch)
       _arguments -C $sourcesS $javadoc $classpath $artifactType $force $keepOptional $mode $ttl $quiet $verbose $progress $maxIterations $repository $sources $noDefault $sbtPluginHack $dropInfoAttr $forceVersion $exclude $scalaVersion $intransitive $classifier $defaultConfiguration $parallel $checksum $benchmark $tree $reverseTree $profile $typelevel $cache
       ;;
+
+    install)
```

Comment:
I could've added parameters completions also for other commands. Since it's a tedious work I decided to go with partial change